File: Dice.java
import java.util.Random;
import java.util.Scanner;



class Dice{
    public static void main(String[] args){
	int saikoro1;
	int saikoro2;
	Scanner sc = new Scanner(System.in);

	Random random = new Random();
	saikoro1 = random.nextInt(6)+1;
	saikoro2 = random.nextInt(6)+1;
        int sum = saikoro1 + saikoro2;

	
	System.out.println("What is your name?");
        String name = sc.next();
	System.out.println("Hello, " + name + "!");


	
	System.out.println("Rolling dice...");
	System.out.println("Die 1: " + saikoro1);
        System.out.println("Die 2: " + saikoro2);
        System.out.println("Total value: " + sum);
        
	

    } 

}

File: Dice.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>


int main(void){
 
  printf("Rolling dice..\n");
  srand(time(NULL));
  
  int D1 = rand() % 6 +1;
  
  int D2 = rand() % 6 + 1;
  
  int sum = 0;


  sum = D1+D2;

  printf("Die 1: %d\n",D1);
  printf("Die 2: %d\n",D2);
  printf("Total value: %d\n",sum);

  return 0;

}

